Frontend/CO.CDP.OrganisationApp/ThirdPartyApiClients/CompaniesHouse/CompaniesHouseApi.cs

```diff
@@ -46,14 +51,24 @@ public class CompaniesHouseApi(IConfiguration configuration,
                 .GetAsync()
                 .ReceiveJson<CompanyProfile>();
         }
+        catch (FlurlHttpException ex) when ((ex.StatusCode == (int)HttpStatusCode.NotFound) || (ex.StatusCode == (int)HttpStatusCode.InternalServerError))
+        {
+        }
         catch (Exception exc)
         {
-            logger.LogError(exc, "Failed during call to get company profile.");
+            Log(exc, companyNumber);
         }
 
         return profile;
     }
 
+    private void Log(Exception exc, string companyNumber)
+    {
+        var logException = new ExceptionLogging($"Failed to call Companies House API for company number: {companyNumber}.", ErrorCodes.CompaniesHouseApiError, exc);
```

Comment:
`companyNumber` is assigned an error code, instead of the company number.

Reply:
The second param to ExceptionLogging is the error code, not the company number for the string.

Reply:
The message only has one placeholder, and it's for the company number.